src/app/components/data/shared/rendering-container-image-picker/rendering-container-image-picker.component.ts

```diff
@@ -0,0 +1,151 @@
+import {
+    ChangeDetectionStrategy, ChangeDetectorRef, Component, Input,
+    OnChanges, OnDestroy, forwardRef,
+} from "@angular/core";
+import {
+    ControlValueAccessor,
+    FormControl,
+    NG_VALUE_ACCESSOR,
+} from "@angular/forms";
+
+import { RenderApplication, RenderEngine, RenderingContainerImage } from "app/models/rendering-container-image";
+import { RenderingContainerImageService } from "app/services";
+import { BehaviorSubject, Observable, Subject, combineLatest } from "rxjs";
+import { map, switchMap, takeUntil } from "rxjs/operators";
+import "./rendering-container-image-picker.scss";
```

Comment:
blank line before

Reply:
done (I think , duplicate of above comment?)

Reply:
done